From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

This test suite needs a live git repository to work correctly.

diff --git a/liquid-fixpoint.cabal b/liquid-fixpoint.cabal
index e90a0e3b..ff530d31 100644
--- a/liquid-fixpoint.cabal
+++ b/liquid-fixpoint.cabal
@@ -182,6 +182,7 @@ executable fixpoint
   default-language: Haskell98
 
 test-suite test
+  buildable:        False
   import: warnings
   type:             exitcode-stdio-1.0
   main-is:          test.hs
